plugins/events-backend-module-http/src/validator/types.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Request } from 'express';
+
+/**
+ * Details for how to respond to the rejection
+ * of the received HTTP request transmitting an event payload.
+ *
+ * @public
+ */
+export interface RequestRejectionDetails {
+  status: number;
+  payload: any;
```

Comment:
Actually, `as any` is not needed as it can be assigned to `any` and the only place using it (accepting `any`), therefore does not care about this change at all. I will still keep this change.